### DISCUSSION TITLE
Unverify users

### DISCUSSION THREAD

**Discussion by flashedk** [pocketbase/pocketbase#6156] [Q&A] [answered]
For a yearly refresh of the userbase (students who might not be part of university anymore), i'm looking for a way to mass-unverify the users table. Can I do this via .update()?

**Comment by chnfyi** [comment]
To help you we need more information:
- Which field/criteria is supposed to help you know that a student should not be part of the university anymore ?
- What do you mean exactly by _unverify the users_ ?

**Reply by flashedk** [reply]
Thanks. I want to set the "verified"-field to false, so that on Login they have to send a new verification mail to the university mail adress.

**Reply by chnfyi** [reply, marked as answer]
Here is a jsvm example for a yearly update (at January 1st) of the `verified` field to `false` for _all users_ from the `users` collection. If you want to update only specific users based on some criterias you can add dbx expressions as optional parameter to the `findAllRecords` method, see documentation [here](https://pocketbase.io/docs/js-records/#fetch-multiple-records).

```javascript
cronAdd("yearly-unverify", "0 0 1 1 *", () => {
    const records = $app.findAllRecords("users")
    for (let record of records) {
        record.setVerified(false)
        $app.save(record)
    }
}
```

**Reply by flashedk** [reply]
Thanks, that worked lika a charm. I just didn't know if you can set the 'verified' field through .update() because I didn't find it in the documentation. But yes, you can.

**Reply by chnfyi** [reply]
`verified` field is a **system** field, same for `id`, `email`, `emailVisibility`, `password` and `tokenKey` fields. You can update them only as a superuser.
